src/client.js

```diff
@@ -333,7 +333,7 @@ class Client extends Base {
           result = obj.Body[outputName];
         }
         if (!result) {
```

Comment:
some how there are two if blocks implementing the same condition

https://github.com/strongloop/strong-soap/blob/92239c41e2135eeab3c3ae0ec2e4f46253558945/src/client.js#L330-L334

https://github.com/strongloop/strong-soap/blob/92239c41e2135eeab3c3ae0ec2e4f46253558945/src/client.js#L335-L341

along with the changes in this PR we could fix the first block with the regex as well 
`(?:(Out|\.Out)(?:put)?|(Response|\.Response))$`